force-app/main/default/classes/UTIL_OrgTelemetry.cls

```diff
@@ -437,5 +439,32 @@ public without sharing class UTIL_OrgTelemetry {
             TelemetryParameterName.Data_CountLicenseCredentials.name(),
             countLicenseCredentials
         );
+
+        featureManager.setPackageIntegerValue(
+            TelemetryParameterName.Data_CountSeasonalAddresses.name(),
+            [
+                SELECT COUNT()
+                FROM Address__c
+                WHERE
+                    Seasonal_Start_Month__c != NULL
+                    AND Seasonal_Start_Day__c != NULL
+                    AND Seasonal_End_Month__c != NULL
+                    AND Seasonal_End_Day__c != NULL
+            ]
+        );
+
+        featureManager.setPackageIntegerValue(
+            TelemetryParameterName.Data_CountCurrentYearSeasonalAddresses.name(),
+            [
+                SELECT COUNT()
+                FROM Address__c
+                WHERE
+                    Seasonal_Start_Month__c != NULL
+                    AND Seasonal_Start_Day__c != NULL
+                    AND Seasonal_End_Month__c != NULL
+                    AND Seasonal_End_Day__c != NULL
+                    AND CreatedDate = THIS_YEAR
```

Comment:
Looking at the [technical design](https://salesforce.quip.com/EKYnAb2wBrcq#temp:C:VDS14aec93765ec494db999a2a1d), the business value suggests we want records created in the last 12 months even though the name of the feature suggests current year as you've done. If you've confirmed the intended behavior, ignore this - otherwise just want to point out whether we want to utilize`LAST_N_MONTHS:12` vs `THIS_YEAR`

Reply:
Great catch @kg345, `LAST_N_MONTHS:12` seems to exclude the current month - alternatively have plugged in `LAST_N_DAYS:365` ([reference](https://github.com/SalesforceFoundation/EDA/pull/1922/commits/c643b545a9c3cfc1904215b80fde69e8a56aac59)).

Reply:
Good catch @kg345, `LAST_N_MONTHS:12` seems to exclude the current month - alternatively have plugged in `LAST_N_DAYS:365` ([reference](https://github.com/SalesforceFoundation/EDA/pull/1922/commits/c643b545a9c3cfc1904215b80fde69e8a56aac59)).